.gitignore

```diff
@@ -20,3 +20,5 @@ target/
 
 .gdb_history
 .gdbinit
+
+.cache/clangd/index
```

Comment:
Should we just ignore all of `.cache`?

Reply:
I'm not sure.  I just know `clangd` keeps making this file so I'm ignoring it.